### DISCUSSION TITLE
Is there guide on the options for the URL in getting the RSS feed and doing other things?

### DISCUSSION THREAD

**Discussion by signal15** [FreshRSS/FreshRSS#5499] [Q&A]
https://x.x.x.x/i/?a=rss&user=admin&token=MYTOKEN&hours=168

This works fine.  If I wanna mark a favorite, this works also:
https://x.x.x.x/i/?c=entry&user=admin&token=MYTOKEN&a=bookmark&id=1687900504804432

How can I add labels using this method?  Can I add labels that don't already exist?  Can I add multiple labels with a single call?  I cannot find docs on this anywhere.

Also, can I search on labels?  Can I search on more than one label in a single search?

**Comment by Alkarex** [comment]
Hello,
Please note that we have [an API](https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html) for richer client interaction.

Otherwise, you can open your Web browser's network inspection tools to see the different Web calls:
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/8faeba95-4b39-4cb0-b070-3d4d84229eea)

Tagging the entries expects an HTTP POST, and it currently expects a single entry and a single tag:
https://github.com/FreshRSS/FreshRSS/blob/ee195354d96e698a521cf9584e2f4e85190dc5ef/app/Controllers/tagController.php#L31C4-L64
